torchbenchmark/models/phi_1_5/__init__.py

```diff
@@ -0,0 +1,14 @@
+from torchbenchmark.tasks import NLP
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+
+class Model(HuggingFaceModel):
+    task = NLP.LANGUAGE_MODELING
+    # TODO: is this needed given that this is a pre-trained model?
+    # DEFAULT_TRAIN_BSIZE = 6
```

Comment:
Train test can still be added on a pre-trained model. If you are adding a train test, `DEFAULT_TRAIN_BSIZE` must be defined.